pkg/controller/vitessshard/reconcile_tablets.go

```diff
@@ -360,3 +362,45 @@ func tabletAvailableStatus(resultBuilder *results.Builder, pod *corev1.Pod, read
 	resultBuilder.RequeueAfter(tabletAvailableTime - readyDuration)
 	return corev1.ConditionFalse
 }
+
+func (r *ReconcileVitessShard) checkPVCDiskUpdates(ctx context.Context, tabletSpec *vttablet.Spec, pod *corev1.Pod) {
+	// If no PVC is configured for this tablet pod, bail out.
+	if tabletSpec.DataVolumePVCSpec == nil {
+		return
+	}
+
+	pvc := &corev1.PersistentVolumeClaim{}
+	namespacedClaim := apitypes.NamespacedName{
+		Namespace: pod.Namespace,
+		Name:      tabletSpec.DataVolumePVCName,
+	}
```

Comment:
The type `client.ObjectKey` is an alias of `apitypes.NamespacedName` so we can technically use either one. However, to make the codebase as a whole easier to work with, we picked one (`client.ObjectKey`) and defined a convention that we should stick with it.

```suggestion
	pvcKey := client.ObjectKey{
		Namespace: pod.Namespace,
		Name:      tabletSpec.DataVolumePVCName,
	}
```

Reply:
Makes sense, done!